template-minimal/shared/deco.gen.ts

```
@@ -0,0 +1,183 @@
+export interface DECO_CHAT_STATE_VALIDATIONOutput {
+  valid: Boolean;
+}
+
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface DECO_CHAT_VIEWS_LISTInput {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)